src/javascript/common/champion.js

```diff
@@ -97,6 +100,11 @@ const Champion = (function() {
         Header.init();
         ChampionSignup.load();
         Utility.handleActive();
+
+        // Handle login links
+        if (!Client.is_logged_in()) {
```

Comment:
It doesn't need to be called on all pages. Please move it inside `ClientType`. Thanks.